### PR TITLE
Accept static route for transfer request

### PR DESCRIPTION
**Pull request by netcriptus**

### CODE REVIEW

File: raiden/tests/integration/api/rest/test_payments.py [hunk 1]
```diff
@@ -148,6 +148,107 @@ def test_api_payments(
...
+            token_address=to_checksum_address(token_address),
+            target_address=to_checksum_address(target_address),
+        ),
+        json={"amount": "1"},
+    )
+    with watch_for_unlock_failures(*raiden_network):
+        response = request.send().response
+    assert_proper_response(response)
+    json_response = get_json_response(response)
+    assert_payment_secret_and_hash(json_response, payment)
```
Comment (konradkonrad) [inline raiden/tests/integration/api/rest/test_payments.py]: This part of the test has little to do with `test_api_payments_without_pfs` - I suggest to remove it.

File: raiden/tests/integration/api/rest/test_payments.py [hunk 2]
```diff
@@ -148,6 +148,107 @@ def test_api_payments(
...
+        api_url_for(
+            api_server_test_instance,
+            "token_target_paymentresource",
+            token_address=to_checksum_address(token_address),
+            target_address=to_checksum_address(target_address),
+        ),
+        json={"amount": str(deposit)},
+    )
+    response = request.send().response
+    assert_proper_response(response, status_code=HTTPStatus.CONFLICT)
```
Comment (konradkonrad) [inline raiden/tests/integration/api/rest/test_payments.py]: This part of the test would make sense for `test_api_payments_without_pfs` if we would actually provide the `paths` argument and then check that the path validation leads to the correct conflict.

File: raiden/tests/integration/api/rest/test_payments.py [hunk 3]
```diff
@@ -148,6 +148,107 @@ def test_api_payments(
...
+    request = grequests.get(
+        api_url_for(
+            api_server_test_instance, "raideninternaleventsresource", limit=limit, offset=0
+        )
+    )
+    response = request.send().response
+    assert_proper_response(response)
+    events = response.json()
+    assert len(events) == limit
+    assert all("TimestampedEvent" in event for event in events)
```
Comment (konradkonrad) [inline raiden/tests/integration/api/rest/test_payments.py]: Not sure how this is related to `test_api_payments_without_pfs` ?

File: raiden/tests/integration/api/rest/test_payments.py [hunk 4]
```diff
@@ -148,6 +148,107 @@ def test_api_payments(
...
+
+
```
Comment (konradkonrad) [inline raiden/tests/integration/api/rest/test_payments.py:331]: I would like to see some tests for the unhappy cases of `paths` validation.

**Review by konradkonrad — CHANGES_REQUESTED** [review]
looks good code wise, two change requests though:
- please add REST API documentation
- please see my comments about the integration test for `test_api_payments_without_pfs`

File: raiden/tests/integration/api/rest/test_payments.py (hunk 1 above)
Comment (netcriptus) [reply raiden/tests/integration/api/rest/test_payments.py]: ✅

File: raiden/tests/integration/api/rest/test_payments.py (hunk 2 above)
Comment (netcriptus) [reply raiden/tests/integration/api/rest/test_payments.py]: ✅

File: raiden/tests/integration/api/rest/test_payments.py (hunk 3 above)
Comment (netcriptus) [reply raiden/tests/integration/api/rest/test_payments.py]: ✅

File: raiden/tests/integration/api/rest/test_payments.py (hunk 4 above)
Comment (netcriptus) [reply raiden/tests/integration/api/rest/test_payments.py:331]: ✅

**Review by konradkonrad — APPROVED** [review]
lgtm! :shipit:

**Pull request merged**